app/templates/views/accessibility_statement.html

```diff
@@ -47,7 +46,15 @@ <h2 class="heading-medium" id="how-accessible-this-website-is">How accessible th
     We know some parts of this website are not fully accessible:
   </p>
   <ul class="govuk-list govuk-list--bullet">
-    <li>if a team adds template folders, screen readers may announce the page heading in a confusing way</li>
+    <li>the footer with buttons in, on the templates page, sometimes overlaps focused links or checkboxes</li>
+    <li>the visual design of the service navigation doesn't work on smaller screens or at high zoom</li>
```

Comment:
‘service navigation’ sounds like a noun - but I’m not sure it’s clear what the thing is

Reply:
```suggestion
    <li>the visual design of the service navigation does not work on smaller screens or at high zoom</li>
```

Reply:
🤔 

I don’t think we should get too hung up about it.

That said, my first thought was ‘the left-hand nav that contains all the things related to your service’, not this pseudo-breadcrumb-nav-thing 😁 

How about:

```suggestion
    <li>the visual design of the organisation and service breadcrumb links does not work on smaller screens or at high zoom</li>
```

❓